### DISCUSSION TITLE
Attach Backup definition to Project's ressources

### DISCUSSION THREAD

**Discussion by McSlow** [vmware-tanzu/velero#7200] [Community support Q&A] [answered]
Hi,
first of all, Velero is working perfectly fine for us. It's installed in the default "velero" namespace, backed by a minio s3 destination.

Now, the question came up to bundle the backup definitions ( especially `backup` and `schedule`) together with all the other resource definitions of a particular project- lets call it `project_a` here - in a way that selected users and/or ci pipelines can apply the backup definitions by themselves - of course not in the `velero` namespace, but in it's own, restricted namespace (e.g. `project_a`).

A short test adding a backup to a different namespace ( eg `velero backup create project_a_backup [...options...] -n project_a` ) seems not to work. 
The Backup Resource is there ( in the `project_a` namespace), but stays in the "new" state, not being processed (possibly because the crd searches for the velero app in that namespace, which is instead running in the velero namespace).

is there a way to accomplish this? I digged a bit through the documentation and closed issues but found nothing.

**Comment by sseago** [comment, marked as answer]
Velero is a namespaced application. If Velero is installed in the `velero` namespace, then it only looks for CRs in the `velero` namespace. If you had a second velero instance installed in the `velero2` namespace, then you could create Backups in `velero2` and then the second velero would process them, while the first would ignore them. Velero intentionally only looks in its own namespace, since it's possible that velero CRs in another namespace belong to a different velero instance.

**Reply by McSlow** [reply]
Thanks for clarifying!
